grumphp.yml

```diff
@@ -1,61 +0,0 @@
-parameters:
-    phpstan.config: "vendor/phpro/symfony-conventions/phpstan.neon"
-    phpstan.level: 2
-    phpcsfixer2.config: "vendor/phpro/symfony-conventions/.php_cs.dist"
-    phpparser.ignore:
-        - vendor
-        - config
-        - tests/bootstrap.php
-        - src/Kernel.php
-    phpstan.ignore:
-        - "tests/"
-        - "Tests/"
-        - "vendor/"
-        - "var/"
-        - "src/Migrations/"
-        - "spec/"
-grumphp:
-    stop_on_failure: true
-    tasks:
-        paratest:
-            metadata:
-                priority: 100
-        phplint:
-            metadata:
-                priority: 200
-        yamllint:
-            parse_custom_tags: true
-            metadata:
-                priority: 200
-        composer:
-            strict: false
-            no_check_publish: true
-            metadata:
-                priority: 200
-        phpparser:
-            ignore_patterns: "%phpparser.ignore%"
-            visitors:
-                no_exit_statements: ~
-                never_use_else: ~
-                forbidden_function_calls:
-                    blacklist: [var_dump, echo, print, dump, print_r, phpinfo]
-            metadata:
-                priority: 200
-        phpstan:
-            level: "%phpstan.level%"
-            configuration: "%phpstan.config%"
-            ignore_patterns: "%phpstan.ignore%"
-            metadata:
-                priority: 200
-        phpcsfixer2:
-            allow_risky: true
-            cache_file: ~
-            config: "%phpcsfixer2.config%"
-            rules: []
-            using_cache: ~
-            config_contains_finder: true
-            verbose: true
-            diff: false
-            triggered_by: ['php']
-            metadata:
-                priority: 200
```

Comment:
One I always add is the following:
We could make the jira project a configurable param

```
        git_commit_message:
            enforce_capitalized_subject: false
            matchers:
                JiraNr: "/JIRA-([0-9]*)/i"
```

Reply:
yeah, I use one too on my project, but its another format...

Reply:
we could make the regex configurable through params